common/common.h

```diff
@@ -107,6 +107,8 @@ struct gpt_params {
     std::string model_draft          = ""; // draft model for speculative decoding
     std::string model_alias          = "unknown"; // model alias
     std::string model_url            = ""; // model url to download
+    std::string auth_token           = ""; // auth bearer token
+    std::string hf_token             = ""; // HF token
```

Comment:
IMO having 2 variables is quite redundant, since they will have the same value.

My suggestion is to remove `--bearer-token` and `auth_token` since I don't think anyone need this for now. We can add it later when needed.

Reply:
I thought it wise to have it there for clarity, but will remove.

Reply:
This is removed.